eox_core/api/v1/urls.py

```diff
@@ -10,3 +11,8 @@
     url(r'^pre-enrollment/$', views.EdxappPreEnrollment.as_view(), name='edxapp-pre-enrollment'),
     url(r'^userinfo/$', views.UserInfo.as_view(), name='edxapp-userinfo'),
 ]
+
+if getattr(settings, "EOX_CORE_UPDATE_USERS_ENABLE", None):
```

Comment:
Would you consider changing the name to EOX_CORE_ENABLE_UPDATE_USERS?

Reply:
alright, that's better